src/generated/rpg_savesystem.h

```diff
@@ -26,6 +26,28 @@
 namespace RPG {
 	class SaveSystem {
 	public:
+		enum ScreenRpgRt {
```

Comment:
Because they seem to be scene names, maybe scene is a better and shorter name instead of screen.

Reply:
Also find this RpgRt pretty ugly, but no idea how to name it. Screen oder Scene are an option. (and maybe add a comment that this is RPG_RT only).

Reply:
I renamed it to just "Screen". I agree with Fdelapena that Scene is a better name than Screen, but Screen seems to be the terminology used by `RPG_RT`. Also since we call our thing a scene, using the word screen here helps avoid confusion.

Reply:
Actually, the RPG_RT terminology is "scene" as well. (also in RGSS in later versions)

Reply:
Well I guess we need to bikeshed this thing then. Here are some options:

* SceneRPG_RT / scene_rpgrt
* LegacyScene / legacy_scene
* Scene / scene
* Scene / scene - and change Player code / enums to use this

Votes anyone?

Reply:
We use Scene because RGSS calls it Scene ;).

> Scene / scene
My favourite. When you document this as "Only used by RPG_RT, not by EasyRPG Player" (which you already do) I think this is not confusing and fine.

> and change Player code / enums to use this
Another possibility but I'm against this because we have more scenes than RPG_RT which would mean adding custom scene entries to liblcf... not a fan of this.